config/initializers/saml_settings.rb

```diff
@@ -1,2 +0,0 @@
-# frozen_string_literal: true
-SAML_CONFIG = Rails.application.config_for(:saml).freeze
```

Comment:
This moved to config/initializers/override_redirect_to_logging.rb:18 since I needed to use the SAML_CONFIG.